command/alloc_status_test.go

```diff
@@ -141,6 +143,16 @@ func TestAllocStatusCommand_Run(t *testing.T) {
 		t.Fatalf("expected to have 'Modified' but saw: %s", out)
 	}
 
+	if !strings.Contains(out, "Modified") {
+		t.Fatalf("expected to have 'Modified' but saw: %s", out)
+	}
+
+	nodeNameRegexpStr := fmt.Sprintf(`\nNode Name\s+= %s\n`, regexp.QuoteMeta(nodeName))
```

Comment:
If you migrate this test to using `require`, you can use `require.Regexp(regexp.MustCompile(regexpStr), out)`, which provides a nicer error message here. (example in this test: https://github.com/hashicorp/nomad/blob/59dec15a7a0ec8d0cdf59e5c30d7292281c51be0/command/alloc_status_test.go#L236-L237)

Reply:
Updated